CMakeLists.txt

```diff
@@ -155,6 +167,26 @@ if (NOT "${NSYNC_LANGUAGE}X" STREQUAL "c++11X")
 	elseif ("${CMAKE_SYSTEM_NAME}X" STREQUAL "OpenBSDX")
 		include_directories ("${PROJECT_SOURCE_DIR}/platform/openbsd")
 		set (NSYNC_POSIX ON)
+    elseif (ANDROID)
+      # Run string comparisons non-lazily to avoid excessive nesting
+      string(COMPARE EQUAL "${CMAKE_ANDROID_ARCH}" "armv7-a" _armv7_a) 
```

Comment:
`CMAKE_ANDROID_ARCH` can't have value `armv7-a`. Possible values listed here:
* https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH.html

Reply:
I see.  I was reading from the `ANDROID_ARCH` *property* documentation.  Thanks for the correction.

https://cmake.org/cmake/help/latest/prop_tgt/ANDROID_ARCH.html?highlight=android_arch#android-arch

Reply:
Fixed.